day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,26 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
+    toppings << "lettuce"
```

Comment:
This works (and exposes some strange ruby behavior), but when updating attributes of a class, using the `@` to reference them directly when updating is more conventional